lib/Dialect/ModuleAttributes.h

```diff
@@ -48,6 +48,19 @@ void moduleClearBackend(Operation *moduleOp);
 void moduleSetOpenfhe(Operation *moduleOp);
 void moduleSetLattigo(Operation *moduleOp);
 
+// Func attributes for client helpers
+
+constexpr const static ::llvm::StringLiteral kClientEncFuncAttrName =
+    "client_enc_func";
+constexpr const static ::llvm::StringLiteral kClientDecFuncAttrName =
+    "client_dec_func";
+
+// The name of the function this client helper is made for.
+constexpr const static ::llvm::StringLiteral kClientHelperFuncName =
+    "func_name";
+// The argument or operand index the client helper function is for.
+constexpr const static ::llvm::StringLiteral kClientHelperIndex = "index";
+
```

Comment:
Bit surprised that funcOp itself allows attr name with no `.` in it. Still, maybe we can get separate it into namespaces like `client.enc_func` so other passes can tell whether this attr is their concern?

As for `func_name` and `index`, better describe that they are inside the dictionAttr like an example here

```mlir
client.enc_func = {func_name = "name", index = 0 : i64}
```

Reply:
Done